xline/src/main.rs

```diff
@@ -521,19 +527,36 @@ async fn main() -> Result<()> {
     let server_addr = server_addr_str
         .iter()
         .map(|addr| {
-            addr.to_socket_addrs()?
+            // TODO: update this after we support https
+            let address = if let Some(address) = addr.strip_prefix("http://") {
+                address
+            } else {
+                addr
+            };
+            address
+                .to_socket_addrs()?
```

Comment:
The dns resolution could contains multipe ip addresses, I think we should collect them all.

Reply:
Fixed